blas/tpls/KokkosBlas2_syr_tpl_spec_decl_rocblas.hpp

```diff
@@ -28,9 +28,8 @@ namespace Impl {
   const int N           = static_cast<int>(A_is_lr ? A.extent(0) : A.extent(1)); \
   constexpr int one     = 1;                                                     \
   const int LDA         = A_is_lr ? A.stride(0) : A.stride(1);                   \
-  rocblas_fill fillMode = (uploChar == 'L' || uploChar == 'l')                   \
-                              ? rocblas_fill_lower                               \
-                              : rocblas_fill_upper;
+  rocblas_fill fillMode = (*uplo == 'L' || *uplo == 'l') ? rocblas_fill_lower    \
```

Comment:
> Shouldn't (*uplo)[0] be the thing to be compared to the characters 'L' and 'l', since uplo is a array of characters?

@eeprude either `uplo[0]` or `*uplo` provide identical comparisons to  L or l, just to reassure I confirmed with a build with rocblas enabled that this will compile and pass. Since this has passed CI, would it be okay to merge this as is, with a follow-up PR with the consistency updates you mentioned?

Reply:
Totally fine, Nathan. Just wanted to make sure we are on the same page. Thank you.